### ISSUE TITLE
Add this package to composer

### ISSUE THREAD

**Issue by iNilo** [rossengeorgiev/vdf-parser#6]
It would be nice to have this added to packagist.org

**Comment by rossengeorgiev** [comment]
Hi, 
I am not familiar with packagist.org

Can you give me an example of importing the library is some php code?

Thanks

**Comment by kbkk** [comment]
Usually PHP applications using Composer also use its autoloader. In this case the file vdf.php will be autoloaded. However this is polluting the global namespace (you can't do exports like JS) so I'd wrap it in a class.

**Comment by rossengeorgiev** [comment]
From what I've looked briefly, there are namespaces and such, so this PR needs more work.

**Comment by kbkk** [comment]
Yeah, this would need to follow some PSR and have its own repository or at least branch so people don't download unnecessary files.

**Comment by rossengeorgiev** [comment]
Does it download all files? Can I put the php stuff in a separate directory?

**Comment by kbkk** [comment]
Seems like it will always download the whole repository branch/tag. So the best option would be to open another repository for the PHP version.
